packages/query/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "outDir": "./lib",
     "rootDir": "./src",
     "strict": true,
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
```

Comment:
Do we need `experimenalDecorators` here? I guess this package do not use decorators :)

Reply:
I don't know, just added a comma) Removed